capellambse/model/crosslayer/__init__.py

```diff
@@ -30,6 +30,22 @@ class BaseArchitectureLayer(c.GenericElement):
     all_classes = c.ProxyAccessor(
         information.Class, deep=True, aslist=c.ElementList
     )
+    all_collections = c.ProxyAccessor(
+        information.Collection, deep=True, aslist=c.ElementList
+    )
+    all_unions = c.ProxyAccessor(
+        information.Union, deep=True, aslist=c.ElementList
+    )
+    all_enumerations = c.ProxyAccessor(
+        information.Enumeration, deep=True, aslist=c.ElementList
+    )
+    all_complexvalues = c.ProxyAccessor(
```

Comment:
"Complex Value" are two words, and the class name is CamelCased, so it makes sense to name this attribute `all_complex_values`.

```suggestion
    all_complex_values = c.ProxyAccessor(
```